src/referencestrip.js

```diff
@@ -455,7 +455,7 @@ function loadPanels( strip, viewerSize, scroll ) {
                 animationTime:          0,
                 loadTilesWithAjax:      strip.viewer.loadTilesWithAjax,
                 ajaxHeaders:            strip.viewer.ajaxHeaders,
-                useCanvas:              strip.useCanvas
+                drawer:                 strip.viewer.drawerOptions.constructor,
```

Comment:
Shouldn't this be `strip.viewer.drawer`? 

Also, it could be an array... Should we duplicate the array so they aren't both references to the same data? Or does that even matter after the initial construction? Probably not worth bothering with.

Reply:
How it worked was that when a drawer implementation was picked, the constructor was added to `viewer.drawerOptions`. However, I realize it is more straightforward to access it via `viewer.drawer.constructor` so I've changed it to that.

The `options.drawer` passed to the `Viewer` constructor can be a `string`, `Object (constructor specifically)` or an `Array` of either of those. However, only the main drawer needs to go through the process of determining compatibility. Sub-viewers such as the reference strip should just use the same type as the main viewer. So, no need for the array to be passed through, just the constructor to use.

Reply:
Nice! Might be worth a comment here to make that clear?

Reply:
Added a brief inline comment.